tests/test_grassmannian.py

```diff
@@ -6,12 +6,48 @@
 import geomstats.tests
 
 from geomstats.geometry.grassmannian import Grassmannian
+from geomstats.geometry.grassmannian import GrassmannianCanonicalMetric
+
+Pxy = gs.diag([1., 1., 0.])[0]
```

Comment:
no uppercase letters in variable names, it is a convention chosen so that developers only have to remember variable names and not which letter is uppercase or not